tests/functional/test.cli.run.js

```diff
@@ -47,4 +48,31 @@ describe('web-ext run', () => {
            }
          });
        }));
+
+  it('should not accept: --target INVALIDTARGET',
+     () => withTempAddonDir(
```

Comment:
In this test, the program is supposed to exit early due to invalid command line parameters. Therefore it is not necessary to create a temporary add-on. You can simplify the test:

```
it('should not accept: --target INVALIDTARGET', async () => {
  const argv = [ ... ] // 'run' and the --target options only
  return execWebExt(argv, {}).then(({exitCode, stderr}) => {
    ...
  });
});